### PR TITLE
Invite group to project issue #770

### PR DESCRIPTION
**Pull request by SleeplessRegina**
https://github.com/DemocracyLab/CivicTechExchange/issues/770
The confirmation modal submit function is not returning a promise like the ConfirmationModal expects. 
Used the promiseHelper class and the spinning and unclickable "Yes" button is resolved. 
Screen Shot 2023-03-27 at 22 45 40

### CODE REVIEW

**Review by PeterBreen — APPROVED** [review]
This looks good; I tested it in my dev environment and it worked correctly. I'd like to test on one of our staging environments as soon as we resolve #956 before merging into main branch, but that's mostly a formality.

**Comment by SleeplessRegina** [comment]
> This looks good; I tested it in my dev environment and it worked correctly. I'd like to test on one of our staging environments as soon as we resolve #956 before merging into main branch, but as it works now, I'm approving.

Thanks for approving. I'll merge into master once the #956 solved.

**Pull request merged**
